IBX-1868: Added information about reference and supplement the missing information (#1547)
ezsystems/developer-documentation: docs/guide/data_migration/data_migration_actions.md

                values:
                    - 1
```

> **Note**
> 
> In action `assign_user_group_to_role` limitation type Section can only use section id
> 
> mode: Update
> ``` yaml
>     actions:
>         -
>             action: unassign_role_user_group
>             id: 1
> ```
> **Note**
> 
> In action `unassign_role_user_group` id is role assignment ID from table `ezuser_role`

Reviewer: ```suggestion
!!! note
    
    In the `assign_user_group_to_role` action, Limitation type Section can only use Section ID:
    
    mode: Update
    
    ``` yaml
        actions:
            -
                action: unassign_role_user_group
                id: 1
    ```
    
    In the `unassign_role_user_group` action, the ID is Role assignment ID from the `ezuser_role` table.
```

Reviewer: The note content needs to be indented for the formatting to work, for now it's broken: 
![image](https://user-images.githubusercontent.com/13621049/156746550-06222bbb-1683-4c38-9c17-debc38bb27f6.png)

Author: now should be correct
